### PR TITLE
PLAT-128528: Update tutorial documentations

### PR DESCRIPTION
**Pull request by YunhoLee0**
Enact-DCO-1.0-Signed-off-by: Yunho Lee (yunho2.lee@lge.com)

Merge feature/kittenbrowser-migration branch to develop branch

### CODE REVIEW

File: src/pages/docs/tutorials/tutorial-kitten-browser/data-and-state/index.md [hunk 1]
```diff
@@ -167,10 +171,26 @@ Instead, you'll most often apply these using the [rest and spread operators](../
...
 
+As well as wrapping your component with `Spottable`, we need to define the colors that will be used for the Spotlight effect when the `Kitten` is focused. We'll use the `focus` mixins provided by Sandstone to set the background color and text color of the component. First, import the mixins into `./src/components/Kitten/Kitten.module.less` which defines the style of the `Kitten` component. Then, use `focus` mixins to define the desired color properties. We'll use `lightgrey` as the background color and `black` as the color of the text when the component is focused.
+
+**./src/components/Kitten/Kitten.module.less**
+```css
+@import "~@enact/sandstone/styles/mixins.less";
+
+.kitten {
+	display: inline-block;
+	padding: 22px;
```
Comment (MikyungKim) [inline src/pages/docs/tutorials/tutorial-kitten-browser/data-and-state/index.md:182]: Can we make this as multiples of 6?

Comment (MikyungKim) [reply src/pages/docs/tutorials/tutorial-kitten-browser/data-and-state/index.md:182]: 24px would be good.

File: src/pages/docs/tutorials/tutorial-kitten-browser/lists/index.md [hunk 2]
```diff
@@ -75,7 +75,7 @@ If you've been running the app as we go, you likely noticed a couple issues afte
 ```css
 .kitten {
 	display: inline-block;
-	padding: 12px;
+	padding: 22px;
```
Comment (MikyungKim) [inline src/pages/docs/tutorials/tutorial-kitten-browser/lists/index.md:78]:
```suggestion
	padding: 24px;
```

File: src/pages/docs/tutorials/tutorial-kitten-browser/data-and-state/index.md (hunk 1 above)
Comment (YunhoLee0) [reply src/pages/docs/tutorials/tutorial-kitten-browser/data-and-state/index.md]: OK

File: src/pages/docs/tutorials/tutorial-kitten-browser/lists/index.md (hunk 2 above)
Comment (YunhoLee0) [reply src/pages/docs/tutorials/tutorial-kitten-browser/lists/index.md]: OK

**Review by MikyungKim — APPROVED** [review]
LGTM

**Review by seunghoh — APPROVED** [review]

File: src/pages/docs/tutorials/tutorial-hello-enact/adding-css/index.md [hunk 3]
```diff
@@ -22,7 +22,7 @@ to components using the `className` property.
 
 > You might expect JSX to use `class` to mirror HTML but that isn't possible since JSX is ultimately
 > transpiled to JavaScript, in which `class` is a reserved word. See [DOM
-> Differences](https://facebook.github.io/react/docs/dom-differences.html) for other instances where
+> Differences](https://reactjs.org/docs/dom-elements.html) for other instances where
```
[the remark this answers is not in the document]
Comment (YunhoLee0) [reply src/pages/docs/tutorials/tutorial-hello-enact/adding-css/index.md:25]: Thank you. I'll change it when I update the kitten browser documentation. (PLAT-124073)

**Pull request merged**